OpenRA.Mods.Common/Activities/FindAndDeliverResources.cs

```diff
@@ -56,7 +59,7 @@ protected override void OnFirstRun(Actor self)
 				// We have to make sure the actual "harvest" order is not skipped if a third order is queued,
 				// so we keep deliveredLoad false.
 				if (harv.IsFull)
-					QueueChild(new MoveToDock(self));
+					QueueChild(new MoveToDock(self, linkClient, linkLineColor: linkClient.LinkLineColor));
```

Comment:
Do we need a separate `linkLineColor` param if we can just grab it from the `linkClient` anyway? (Haven't checked other callers so sorry if I'm wrong here)

Reply:
in #20667 we want no color for some orders, specifically for lua ones

Reply:
the commit is called `Add support for not displaying target lines`